db/006-reports.sql

```diff
@@ -0,0 +1,26 @@
+CREATE SCHEMA IF NOT EXISTS reports;
+
+CREATE TABLE IF NOT EXISTS reports.idf_station_timeseries (
```

Comment:
Would be nice to point to the struct documentation here, in case someone reaches this first and is confused or assumes there is no documentation.

Reply:
Done!